Keas.Core/Domain/Key.cs

```diff
@@ -9,6 +9,7 @@ public class Key : AssetBase {
         public string SerialNumber { get; set; }
 
         public string LockId { get; set; }
+        public Room Room { get; set; }
```

Comment:
@laholstege this won't be the case that a key has one room associated, maybe not even any via this mechanism.